src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -37,7 +37,7 @@ public sealed class LegacyPackageReferenceProject : PackageReferenceProject<Dict
     {
         private readonly IVsProjectAdapter _vsProjectAdapter;
         private readonly IVsProjectThreadingService _threadingService;
-
+        private const bool IsSdk = false; // default value for non sdk projects
```

Comment:
Personally, I don't see a big value in making this a const, when it's only used once.